### PR TITLE
docs: Updated tables_client docstring

### LINKED ISSUE #99: incorrect docstrings re: type_code (Tables)
This example snippet (and possibly many others) are incorrect, as `type_code` no longer takes a string.

https://github.com/googleapis/python-automl/blob/master/google/cloud/automl_v1beta1/services/tables/tables_client.py#L1264

### PR DESCRIPTION
**Pull request by kweinmeister**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-automl/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #99 🦕

### CODE REVIEW

**Review by nicain — APPROVED** [review]
LGTM

File: google/cloud/automl_v1beta1/services/tables/tables_client.py [hunk 1]
```diff
@@ -38,12 +38,12 @@
 def to_proto_value(value):
     """translates a Python value to a google.protobuf.Value.
 
-  Args:
-  value: The Python value to be translated.
+    Args:
+    value: The Python value to be translated.
```
Comment (busunkim96) [inline google/cloud/automl_v1beta1/services/tables/tables_client.py:42]:
```suggestion
    Args:
        value: The Python value to be translated.
```

File: google/cloud/automl_v1beta1/services/tables/tables_client.py [hunk 2]
```diff
@@ -38,12 +38,12 @@
...
 
-  Returns:
-  Tuple of the translated google.protobuf.Value and error if any.
-  """
+    Returns:
+    Tuple of the translated google.protobuf.Value and error if any.
```
Comment (busunkim96) [inline google/cloud/automl_v1beta1/services/tables/tables_client.py:45]:
```suggestion
    Returns:
        Tuple of the translated google.protobuf.Value and error if any.
```

**Review by busunkim96 — APPROVED** [review]
Indentation nits to match https://google.github.io/styleguide/pyguide.html#383-functions-and-methods, otherwise LGTM.

**Pull request merged**